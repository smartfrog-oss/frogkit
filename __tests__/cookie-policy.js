import { mount } from 'vue-test-utils'
import CookiePolicy from '@/components/cookie-policy'

describe('it should render and match snapShot', () => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(CookiePolicy)
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('CookiePolicy should emit close event when close icon is clicked', () => {
    const stub = jest.fn()
    const wrapper = mount(CookiePolicy)
    wrapper.vm.$on('close', stub)
    wrapper.vm.$el.querySelector('.cookie-policy__close').click()
    expect(stub).toBeCalled()
  })
})
