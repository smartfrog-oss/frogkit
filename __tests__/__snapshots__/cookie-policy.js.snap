// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render and match snapShot should render component and match snapshot 1`] = `
<flex justify="space-between" class="cookie-policy">
  <p class="cookie-policy__content">Durch die Nutzung dieser Website stimmst du der Verwendung von Cookies zu. Weitere Informationen und Hinweise zur Ã„nderung deiner Cookie-Einstellungen findest du in unseren <a href="#">Cookie-Richtlinien</a>.</p>
  <div class="cookie-policy__close">
    <icon color="white" icon="close"></icon>
  </div>
</flex>
`;
